src/modules/Login/action.js

```diff
@@ -103,7 +103,15 @@ export const initializeAccount = (account: EdgeAccount, touchIdInfo: Object) =>
       if (edgeWallet.type === 'wallet:ethereum') {
         if (state.ui.wallets && state.ui.wallets.byId && state.ui.wallets.byId[walletId]) {
           const enabledTokens = state.ui.wallets.byId[walletId].enabledTokens
-          edgeWallet.enableTokens(enabledTokens)
+          const customTokens = state.ui.settings.customTokens
```

Comment:
So, I am fine with this, however to be consistent should you create a getter similar to everything we do for getting ui settings out of redux - .. a `SETTINGS_API.getCustomTokens(state)` instead of the direct path to the piece of state `state.ui.settings.customTokens`.  I do see in this function that other pieces of state are accessed directly. so it is consistent within this function... So this is not a formal change request, just wondering if we should change it. Thoughts?

Reply:
I'll bring this up at next engineering meeting. I've found getters to make debugging more of a headache

Reply:
I believe we actually mentioned to avoid abstracting the Redux store and to access it directly to avoid unnecessary code and boilerplate work. We also get better Flow coverage that way.